Create Project_taxonomy_recommendations.md (#7)
numfocus/disc-unconference-2023-projects: unconference/Project_taxonomy_recommendations.md

# Group 3: Project taxonomy and recommendations for Diversity Equity and Inclusion

> ## Working Group Members:

Reviewer: see the top of https://github.com/numfocus/disc-unconference-2023-projects/blob/main/unconference/mentoring.md for some info we should add that would include the author info below

* Katrina Riehl (@kriehl)
* Madicken Munk (@munkm)
* Rafal Malanij (@elgravel)
* Jenna Swarthout Goddard (@jennaswa)
* Monica Granados (@monsauce) 
* Prabhant Singh (@prabhant)
* Tania Hernandez (@TaniaHernandez23)
* Daniel S. Katz (@danielskatz)

Emeritus Working Group Members:

* Cheuk Ting Ho

Roles: 

* Lead
    * Tania 
* Note taker(s)
    * Jenna
* Understudy Notetaker: 
    * Madicken 
* Reporter/Socializer 
    * Monica
* GitHub: https://github.com/numfocus/disc-unconference-2023-projects

> # Goals

Reviewer: ```suggestion
# Goal
```

        * Governance
            * Leadership
            * Committees
            * Roles and responsibilities
            * Decision-making
            * Conflict resolution
            * Staffing
            * Funding and finances
            * CoC (policies, enforcement, EXISTENCE)
        * Community
            * People management
            * Engaging with users/contributors/developers/maintainers
            * Mentorship (link to mentorship group’s work)
            * Training
            * Marketing
            * Communication channels
        * Technical (Infrastructure)/Technology
            * Documentation
            * Roadmap
            * Hardware (e.g., OS)
            * Software (e.g., repo, CI/CD pipelines, quality/security measures, webhooks)
            * Access management
            * Release practices 
* Stages of project life cycle/timeline

# Next steps:

* Define language and make consistent throughout the document
* Pull out general themes that apply throughout
* Ensure recommendations truly relate back to DEI and are actionable items
* Assess how recommendations support openness/transparency and can be done in a sustainable manner

> * Build out as a matrix of how recommendations apply to projects at different life stages

Reviewer: ```suggestion
* Build out a matrix of how recommendations apply to projects at different life stages
```

* Find citations/references to evaluate where diversity is missing

# Resources (from Slack)

* [https://github.com/badging/project-diversity-and-inclusion](https://github.com/badging/project-diversity-and-inclusion)
* [https://doctorow.medium.com/the-title-card-from-sumana-harihareswaras-github-talk-what-would-open-source-look-like-if-it-were-35b0360f8b89](https://doctorow.medium.com/the-title-card-from-sumana-harihareswaras-github-talk-what-would-open-source-look-like-if-it-were-35b0360f8b89)
* [https://www.jmlr.org/mloss/mloss-info.html](https://www.jmlr.org/mloss/mloss-info.html)

> * [https://goblin.tools/](https://goblin.tools/) Goblin Tools: Break down to-do list items into more granular tasks

Reviewer: ```suggestion
* [Goblin Tools: Break down to-do list items into more granular tasks](https://goblin.tools/) 
```

* Regulatory compliance
    * Definition: Regulatory compliance is how the software, project, and community adheres to the legal constructs that govern it. The legal frameworks in which a project operates may be local, national, or international. 
    * Components:
        * Licenses of upstream software
        * National laws governing user privacy, data sharing and dissemination
        * Export control laws
    * Recommendations:
        * Be aware that regulations vary from country/region, think about how your project will work in a country/region outside where the dominant groups of the project is located
        * Consider how laws will affect major components such as data, hardware and software
        * Prioritize the data privacy of your users particularly those from marginalized groups who can be deidentified. Even data that is anonymised can reveal information particularly when there are few people from a non dominant group 
        * Be aware that your software might only be available in certain regions due to the regulatory framework of various nations/regions. This may limit your community. 
* Trademarking names and logos

>     * Definition: A type of intellectual property consisting of a recognizable sign, design, or expression that identifies the project from a particular source and distinguishes them from others.[^2] A logo is a type of trademark.

Reviewer: ```suggestion
    * Definition: A type of intellectual property consisting of a recognizable sign, design, or expression that identifies the project from a particular source and distinguishes them from others.[^2] A logo is an example of an object that be trademarked.
```

            * Technical infrastructure - see below, noting that some may have costs and some may not
                * May include physical infrastructure (must be purchased and then maintained) as well as services (can have regularly scheduled expenses)
    * Recommendations:
        * Consider the sources of resources; some may not match the projects principles and should not be accepted
        * Consider sharing proposals, perhaps and their reviews, to allow broader communities understand them and use them as examples, in order to broader the set of potential proposers
        * Consider making both the principles/processes and the decisions you make regarding projects resources (both financial and volunteer) and project resource usage (both financial and volunteer) public 
        * Consider principles of DEI in making decisions about resource allocation
        * Support the LMIC group’s work (link to outputs)
* Code of Conduct (CoC)
    * Definition: A code of conduct is a set of rules outlining the norms, rules, and responsibilities or proper practices of an individual party, project, or an organization, and the means to enforce it.
    * Components:

>         * Code of Conduct document which includes defined goals, principles, policies, and processes

Reviewer: ```suggestion
        * Code of Conduct document that includes defined goals, principles, policies, and processes
```

    * Components:
        * Training
        * Task management (could be delegation)
        * Meeting deadlines
        * Conflict resolution
        * Building culture
        * Communication
        * Recognition 
        * Growth (personal, professional) 
        * Mentorship/coaching
    * Recommendations:
        * Consider providing resources for project leadership on how to manage a diverse team 
        * Transparent task management system that allows people to state task assignments, preferences, and progress
            * Tasks can be term-limited or ongoing 
        * Make invisible work into visible tasks, if possible to recognize work of people that may be outside of traditional visible work. Many underrepresented people end up doing invisible work to support their community and this provides some equity into recognition of tasks 
            * When a person does something that they think is not currently listed as a task, they should be able to record it 
        * Be aware of your team’s personal preferences and distribute the work based on that (project is not just code!)
        * Link to volunteer burnout Unconference group
* Stakeholders

> * Definition: Stakeholders can be in one or more of the following categories

Reviewer: ```suggestion
    * Definition: Stakeholders can be in one or more of the following categories
```

>     * Users

Reviewer: ```suggestion
        * Users
```

>     * Contributors

Reviewer: ```suggestion
        * Contributors
```

>     * Developers 

Reviewer: ```suggestion
        * Developers 
```

>     * Maintainers

Reviewer: ```suggestion
        * Maintainers
```

>     * Funders/sponsors

Reviewer: ```suggestion
        * Funders/sponsors
```

>     * External formal or informal media/communications/writers

Reviewer: ```suggestion
        * External formal or informal media/communications/writers
```

>     * Employers of any of the above categories

Reviewer: ```suggestion
        * Employers of any of the above categories
```

> * Recommendations

Reviewer: ```suggestion
    * Recommendations
```

>     * Listed elsewhere in the document

Reviewer: ```suggestion
        * Listed elsewhere in the document
```

> * Engaging with Stakeholders 

Reviewer: ```suggestion
    * Engaging with Stakeholders 
```

        * Formal and informal reporting to funders/sponsors that include DEI aspects on the schedule that they request
        * Be time conscious, make the best use of everyone’s time. 
        * When scheduling, be aware of time zones putting people outside of working hours and be aware of religious holidays, including religions outside of the dominant group of the project.
        * Have different methods of recognition for stakeholders so that there are multiple pathways individuals can be recognized for their work and so that different stakeholders can recognize it. This helps improve visibility of work, including invisible work from non-dominant groups. 
        * Try to have stakeholders from different groups (or all stakeholders) represented on your committees and in your project leadership. 
        * Provide opportunities beyond code contribution for stakeholders to contribute to the project that match the preferences and styles of different people. 
        * Offer as many community resources in as many languages and as many accessible formats as possible

>         * Link to DISCOVER cookbook

Reviewer: ```suggestion
        * Link to [DISCOVER cookbook](https://discover-cookbook.numfocus.org/intro.html)
```

        * Link to contributor diversity tooling Unconference group
* Mentorship 
    * Definition: Guidance, or direction given to contributors by more established contributors or leadership in the service of having less experienced contributors have a successful experience and growth in the project.

>     * Definition 2: A relationship between two people that helps both of them grow via regular communication. Typically one is the mentor who has particular skills and experiences, and the other is the mentee, who has a particular goal for the relationship.

Reviewer: Do we want to pick one of these 2 definitions, or merge them? having 2 in this document probably doesn't make sense.

# Group 3: Project taxonomy and recommendations for Diversity Equity and Inclusion
```{article-info}
:avatar: images/disc-icon.jpg
:avatar-outline: muted
:author: "[@kriehl](https://github.com/kriehl), [@munkm](https://github.com/munkm), [@elgravel](https://github.com/elgravel), [@jennaswa](https://github.com/jennaswa), [@monsauce](https://github.com/monsauce), [@prabhant](https://github.com/prabhant), [@TaniaHernandez23](https://github.com/TaniaHernandez23), [@danielskatz](https://github.com/danielskatz)"
:class-container: sd-p-2 sd-outline-muted sd-rounded-1

> 

Author: ```

> 

Author: ```suggestion
```
```
